integrations/prodigy_openai/project.yml

```diff
@@ -0,0 +1,119 @@
+title: "Benchmarking OpenAI datasets"
+
+directories:
+  - "assets"
+  - "configs"
+  - "corpus"
+  - "metrics"
+  - "openai"
+  - "scripts"
+  - "training"
+
+vars:
+  gpu_id: 0
+  config: "ner.cfg"
+  prompt_template: "ner_prompt.jinja2"
+  seed: 42
+
+assets:
+  - dest: "assets/span-labeling-datasets"
+    description: "The span-labeling-datasets repository that contains loaders for AnEM"
+    git:
+      repo: "https://github.com/explosion/span-labeling-datasets/"
+      branch: "master"
+      path: ""
+
+workflows:
+  ner:
+    - get-dataset
+    - train
+    - evaluate
+  gpt:
+    - openai-preprocess
+    - openai-predict
+    - openai-evaluate
+
+commands:
+  - name: "get-dataset"
+    help: "Preprocess the AnEM dataset"
+    script:
+      - sh -c '(cd assets/span-labeling-datasets && spacy project assets)'
+      - sh -c '(cd assets/span-labeling-datasets && spacy project run anem)'
+      - cp -a ./assets/span-labeling-datasets/corpus/ner/. ./corpus/.
+    deps:
+      - assets/span-labeling-datasets
+    outputs:
+      - corpus/anem-train.spacy
+      - corpus/anem-dev.spacy
+      - corpus/anem-test.spacy
+
+  - name: "train"
+    help: "Train a NER model from the AnEM corpus"
+    script:
+      - >-
+        python -m spacy train
+        configs/${vars.config}
+        --output training/
+        --paths.train corpus/anem-train.spacy
+        --paths.dev corpus/anem-dev.spacy
+        --system.seed ${vars.seed}
+        --gpu-id ${vars.gpu_id}
+    deps:
+      - corpus/anem-train.spacy
+      - corpus/anem-dev.spacy
+    outputs:
+      - training/model-best
+
+  - name: "evaluate"
+    help: "Evaluate results for the NER model"
+    script:
+      - >-
+        python -m spacy evaluate
+        training/model-best corpus/anem-test.spacy
+        --output metrics/scores.json
+        --gpu-id ${vars.gpu_id}
+    deps:
+      - training/model-best/
+      - corpus/anem-test.spacy
+    outputs:
+      - metrics/scores.json
+
+  - name: "openai-preprocess"
+    help: "Convert from spaCy format into JSONL."
+    script:
+      - python -m scripts.convert_to_jsonl corpus/anem-test.spacy corpus/anem-test_texts.jsonl
+    deps:
+      - corpus/anem-test.spacy
+    outputs:
+      - corpus/anem-test_texts.jsonl
+
+  - name: "openai-predict"
```

Comment:
No I haven't :/ Time-wise though I ran the query for an hour and a half, with intermittent stops because of connection errors (`--resume`). I haven't tracked how much (money) it costs.